packages/salesforcedx-vscode-apex-replay-debugger/src/breakpoints/checkpointService.ts

```diff
@@ -336,6 +336,7 @@ export class CheckpointService implements TreeDataProvider<BaseNode> {
               if (deleteResult) {
                 const result = JSON.parse(deleteResult) as BatchDeleteResponse;
                 if (result.hasErrors) {
+                  // What is this message?
                   const errorMessage = nls.localize(
                     'cannot_delete_existing_overlay_action'
```

Comment:
@JimSuplizio , what is this message supposed to be? It doesn't exist in i18n.ts.

Reply:
@DatGuyJonathan , that should be the constant from i18n.ts cannot_delete_existing_checkpoint. It looks like it was overlooked back when messages had overlay